srml/timestamp/src/lib.rs

```diff
@@ -288,7 +288,7 @@ impl<T: Trait> ProvideInherent for Module<T> {
 	}
 
 	fn check_inherent(call: &Self::Call, data: &InherentData) -> result::Result<(), Self::Error> {
-		const MAX_TIMESTAMP_DRIFT: u64 = 60;
+		const MAX_TIMESTAMP_DRIFT: u64 = 30 * 1000;
```

Comment:
I think this refactoring from seconds to millisecons has had one or two of such incidents so far; maybe enough to consider introducing a very basic `enum Time` to make it a _bit harder_ to become inconsistent in the future.

Reply:
also we could suffix the unit to all constant as MAX_TIMESTAP_DRIFT_SECS